component/FloatingActionItem.js

```diff
@@ -33,12 +33,12 @@ class FloatingActionItem extends Component {
   };
 
   renderText() {
-    const { text, position } = this.props;
+    const { text, position, elevation, textElevation, textBackgroundColor, textColor } = this.props;
```

Comment:
Let's add `textElevation`, `textBackgroundColor` and `textColor` inside `FloatingActions.propTypes` and then inside `FloatingActions.defaultProps` so you don't need to specify the default on lines 40 and 41